### DISCUSSION TITLE
해당 프로젝트를 구상하게 된 계기는 무엇인가요?

### DISCUSSION THREAD

**Discussion by kycasdzxc** [josa-moa/josa-moa#21] [Q&A] [answered]
해당 프로젝트를 구상하게 된 계기는 무엇인가요?

**Comment by im-dullin** [comment, marked as answer]
> 해당 프로젝트를 구상하게 된 계기는 무엇인가요?

평소 글 쓰는 것을 즐겨하는데 특정 형식이 있는 글을 서야 할 때에는
조사나 문법에 신경을 많이 써야했습니다.

그러다 문득 조사를 자동으로 정립해주는 프로그램을 만들어볼까 라는 생각이
시초였던 것 같습니다.
